### ISSUE TITLE
Regression: .conversations.get() doesn't not return `.tags` anymore

### ISSUE THREAD

**Issue by s-pace** [santiher/python-helpscout-v2#3]
# Issue: 
Just want to raise a regression wi
```python
from helpscout.client import HelpScout

hs = HelpScout(app_id, app_secret)
conversation = hs.conversations.get(params=params, resource_id=cuid)
```
`conversation` used to have a `.tags` attributes containing the tags of the conversation. This attribute is not present anymore (Since ~2 weeks)

# Alternatives undertaken: 
- `tags` is not present even if params `embed=threads` is present from the URL parameter.

- Using the raw API with `curl` / another client do return the `.tags` array.

It seems that this client is not compliant [with Helpscout documentation](https://developer.helpscout.com/mailbox-api/endpoints/conversations/get/#response). It introduces a *backward incompatibility* 

Let me know if you need any further details and thanks for your tool @santiher

**Comment by santiher** [comment]
Seems like HelpScout modified the API responses when requesting a specific resource id.
I modified the client to reflecting this changes.

You can get version 1.0.1 to fix this, or 2.0.0 to fix this plus calling something like `hs.conversations[14].get()` now returns a single conversation instead of a list with 1 conversation.

Thanks for the heads up.

**Comment by s-pace** [comment]
@santiher thanks for the very quick solution. I will upgrade and update our business logic.

Thanks again.
